mozilla/fxa-email-service
# feat(ses): implement ses-based email sending

Fixes #2.

To test it out:

```
curl -d '{"to":"foo@example.com","subject":"bar","body":{"text":"baz"}}' -H 'Content-Type: application/json' http://localhost:8000/send
```

~~Note that we're currently pinned to the `nightly-2018-04-28` toolchain while we wait on a Rocket update, so you'll need to set that up like so:~~

```
rustup toolchain install nightly-2018-04-28
rustup override set nightly-2018-04-28
```

Also note that using a simple `cargo t` to run the tests will fail. You need to run them in a single thread:

```
cargo t -- --test-threads=1
```

I added a shell script called `t` to make this less annoying, so you just need to type `./t` to run them.

There's a bunch of implementation decisions that I'll call out in the inline comments.

@mozilla/fxa-devs r?
